zap/src/main/dist/lang/Messages_ar_SA.properties

```diff
@@ -1939,6 +1939,8 @@ history.addnote.title                  = \u0625\u0636\u0627\u0641\u0629 \u0645\u
 history.browser.popup                  = Open URL in System Browser
 history.browser.warning                = Failed to display HTTP message in browser.
 history.browser.disabled               = Displaying HTTP messages in browser is disabled as ZAP appears to be running in a container
+history.clear.confirm                  = Clearing all history may also cause Sites Tree nodes to be removed. Do you still want to clear?
+history.clear.tooltip                  = Clear
```

Comment:
Only the `Messages.properties` file needs to be changed.

Reply:
Oops, okay will fix it!